public void testBreakInFinally() {
    try {
        throw new RuntimeException("throw me");
    } finally {
        break;
    }
    Console.log("ok then...");
}
